test/Xunit.StaFact.Tests.Mac/Xunit.StaFact.Tests.Mac.csproj

```diff
@@ -0,0 +1,43 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0-macos</TargetFramework>
+    <OutputType>Exe</OutputType>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>true</ImplicitUsings>
+    <SupportedOSPlatformVersion>10.15</SupportedOSPlatformVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <HasUnitTests>true</HasUnitTests>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="xunit.runner.console" />
+    <PackageReference Include="xunit" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Xunit.StaFact\Xunit.StaFact.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="$(XunitConsoleNetCore2AppPath)" />
+  </ItemGroup>
+
+  <PropertyGroup Label="Globals">
+    <PartialTargetPath>$(RepoRootPath)\bin\Xunit.StaFact.Tests.Mac\$(Configuration)\$(TargetFramework)\osx-x64</PartialTargetPath>
```

Comment:
The `osx-x64` part here seems like it would fail on arm64.

If that's the case, you could do something like the following:

```xml
<HostArchitecture>$([System.Runtime.InteropServices.RuntimeInformation]::OSArchitecture)</HostArchitecture>
<HostArchitecture>$(HostArchitecture.ToLower())</HostArchitecture>
<PartialTargetPath>$(RepoRootPath)\bin\Xunit.StaFact.Tests.Mac\$(Configuration)\$(TargetFramework)\osx-$(HostArchitecture)</PartialTargetPath>
```